layers/+tools/docker/README.org

```diff
@@ -18,6 +18,7 @@ This layer integrates basic container management into Spacemacs.
 - =Docker= build integration
 - =TRAMP= access to running Docker containers
 - =Docker= container and image management via [[https://github.com/Silex/docker.el][docker.el]]
+- =docker-compose= integration via [[https://github.com/Silex/docker.el][docker.el]]
```

Comment:
Not a blocker but could you consider combining these two lines?

Reply:
Er no: 

> = Docker= container and image management, as well as =docker-compose= integration via ...

Reply:
I'd also linkto the [docker engine](https://docs.docker.com/engine) and [docker-compose](https://docs.docker.com/compose) docs.

Reply:
@robbyoconnor Let's do it like this - I will merge this PR and you will improve documentation 😸